tox.ini

```diff
@@ -0,0 +1,27 @@
+[tox]
+minversion = 2.3.1
+envlist = py27dj19
+skipsdist = True
+
+[testenv]
+usedevelop = True
+install_command = pip install -c{env:UPPER_CONSTRAINTS_FILE:https://git.openstack.org/cgit/openstack/requirements/plain/upper-constraints.txt} {opts} {packages}
+setenv = VIRTUAL_ENV={envdir}
+deps = -r{toxinidir}/requirements.txt
+commands =
+  find /opt/dev/atmosphere -type f -name "*.pyc" -delete
+whitelist_externals=
+  find
+  bash
+
+[testenv:py27dj19]
+commands =
+  pip install -U django>=1.9,<1.10
+  pip install factory_boy
+  pip install future
+  pip install vcrpy
+  pip install mock
+  {[unit_tests]commands}
```

Comment:
isn't this more commonly done using the following:
    [tox]
        envlist = {py26,py27}-django{15,16}
    [testenv]
        basepython =
            py26: python2.6
            py27: python2.7
        deps =
            pytest
            django15: Django>=1.5,<1.6
            django16: Django>=1.6,<1.7
            py26: unittest2
        commands = py.test